server/config/local.yaml

```diff
@@ -6,7 +6,7 @@ kafka:
     clientId: eventsgateway
     timeout: 250ms
     brokers: kafka:9092
-    maxMessageBytes: 1000000
+    maxMessageBytes: 5242880
```

Comment:
Is this configured on the docker compose manifest as well?

Reply:
No, only on local.yaml